server/catgenome/src/test/java/com/epam/catgenome/manager/heatmap/HeatmapManagerTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2021 EPAM Systems
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.epam.catgenome.manager.heatmap;
+
+import com.epam.catgenome.entity.heatmap.Heatmap;
+import junit.framework.TestCase;
+import org.apache.lucene.queryparser.classic.ParseException;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.io.IOException;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration({"classpath:applicationContext-test.xml"})
+public class HeatmapManagerTest extends TestCase {
+
+    @Autowired
+    private HeatmapManager heatmapManager;
+
+    @Autowired
+    private ApplicationContext context;
+
+    private String fileName;
+
+    @Before
+    public void setUp() throws IOException, ParseException {
+        this.fileName = context.getResource("classpath:heatmap//Halo_pallet_skmer.tsv").getFile().getPath();
+    }
+
+    @Test
+    public void readHeatmapTest() throws IOException {
+        Heatmap heatmap = Heatmap.builder()
+                .name("test")
+                .path(fileName)
+                .build();
+        heatmapManager.readHeatmap(heatmap);
```

Comment:
Let's add some validation to the test: verify that data is loaded correctly + let's add tests to label/cell annotation as well.